build.gradle

```diff
@@ -257,6 +257,7 @@ configure(javaCodeCheckedProjects) {
         testImplementation 'org.spockframework:spock-core'
         testImplementation 'org.spockframework:spock-junit4'
         testImplementation("org.mockito:mockito-core:3.8.0")
+        testImplementation("org.mockito:mockito-inline:3.8.0")
```

Comment:
This useful addon to Mockito allows for the mocking of static methods.

Reply:
Nice